src/createServer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const { userRouter } = require('./routes/users.js');
+const { expensesRouter } = require('./routes/expenses.js');
+
+const jsonMiddleware = express.json();
+
+function createServer() {
+  const app = express();
+
+  app.use('/users', jsonMiddleware, userRouter);
+  app.use('/expenses', jsonMiddleware, expensesRouter);
```

Comment:
```suggestion
  app.use(express.json())
  app.use('/users', userRouter);
  app.use('/expenses', expensesRouter);
```

Reply:
already done it above, please see fix above